include/deployer.bash

```diff
@@ -242,14 +262,15 @@ main() {
         cmd-export deployer-regenerate regenerate
         cmd-export deployer-delete delete
         cmd-export compose-ps ps
-        cmd-export compose-up start
+        cmd-export compose-up-cmd start
+        cmd-export start-and-migrate-cmd start-and-migrate
```

Comment:
I think we don't need a start and start-and-migrate command exposed, the start itself is sufficient.

If we would like to execute the start without migration then I would recommend a flag e.g.:
cbd start --no-migration